src/CMakeLists.txt

```
@@ -68,6 +69,12 @@ target_link_libraries(_espressopp PUBLIC MPI::MPI_CXX)
 target_link_libraries(_espressopp PRIVATE FFTW3::fftw3)
 target_link_libraries(_espressopp PRIVATE hdf5::hdf5 hdf5::hdf5_hl)
 
+if (SCAFACOS_FOUND)
+    target_compile_definitions(_espressopp PRIVATE -DFCS_EXIST)
```

```suggestion
    target_compile_definitions(_espressopp PRIVATE -DFCS_EXIST -DHAVE_CONFIG_H)
```

Re: @XzzX comment from above